### ISSUE TITLE
vector _pod.tpl does not offer extraContainers 

### ISSUE THREAD

**Issue by ArtemTrofimushkin** [vectordotdev/helm-charts#224]
Is it possible to add `extraContainers` option in [templates/_pod.tpl](https://github.com/vectordotdev/helm-charts/blob/a4e9b7adcf780928fb0a4f5f989c760f75e53d9d/charts/vector/templates/_pod.tpl) (as well as already [existing](https://github.com/vectordotdev/helm-charts/blob/a4e9b7adcf780928fb0a4f5f989c760f75e53d9d/charts/vector/values.yaml#L268) initContainers field) in ability to deploy custom sidecars with vector pods?

**Comment by r0b3r4** [comment]
This one just waiting to be implemented, right?

**Comment by spencergilbert** [comment]
Yeah - I haven't had much time recently to devote to chart features, happy to get a contributor PR in though!

**Comment by r0b3r4** [comment]
Here is the PR https://github.com/vectordotdev/helm-charts/pull/239
